### ISSUE TITLE
Hide Hours field

### ISSUE THREAD

**Issue by josedusol** [jdewit/bootstrap-timepicker#335]
Can i hide the hours field?. I need a "mm:ss" format. Thanks

**Comment by mrhota** [comment]
@josedusol I'm sorry, right now this isn't supported.

It seems like what you want to do is use the timepicker (which lets you pick times of day) as a "duration" picker. Is that right?

**Comment by josedusol** [comment]
That's right. I'm currently using DateTimepicker with custom format. But i prefer your plugin.
